### PR TITLE
Added config for black formatting checks

### PR DESCRIPTION
**Pull request by aditirao7**
Resolves issue #52.

Signed-off-by: Aditi Rao aditirao725@gmail.com

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ python:
   - "3.8"
   - "3.9"
 
+before_install:
+  - pip install black
+  - echo "Black formatting check"
+  - black --check *.py
```
Comment (pombredanne) [inline .travis.yml:12]: Can you make this `-l 100` there too?
100 chars per line is better these days

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks... there is a tiny change needed to be complete

File: .travis.yml (hunk 1 above)
Comment (aditirao7) [reply .travis.yml:12]: > Can you make this `-l 100` there too? 100 chars per line is better these days

Yep, on it.

**Comment by tdruez** [comment]
@aditirao7 Thanks for your contribution, changes merged in https://github.com/package-url/packageurl-python/pull/91

**Pull request closed without merging**